src/main/java/core/basesyntax/model/Car.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+    private static long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(long id, String model, Manufacturer manufacturer) {
+        id++;
```

Comment:
what is that? 
let's remove incrementation from here and `id` from input parameters in the constructor